Streetcode/Streetcode.BLL/MediatR/Sources/SourceLinkCategory/Update/UpdateCategoryCommand.cs

```diff
@@ -5,5 +5,5 @@
 
 namespace Streetcode.BLL.MediatR.Sources.SourceLink.Update
 {
-    public record UpdateCategoryCommand(SourceLinkCategoryDTO Category) : IRequest<Result<Unit>>;
+    public record UpdateCategoryCommand(UpdateSourceLinkCategoryDTO Category) : IRequest<Result<UpdateSourceLinkCategoryDTO>>;
```

Comment:
UpdateSourceLinkCategoryDTO or SourceLinkCategoryUpdateDTO? Same for every DTO that starts with Update/Create

Reply:
SourceLinkCategoryDTO - it's an old variant of respond/respond DTO and it had been before I merged my branch with last changes from develop. Now current variant is with UpdateDTO and all last unit tests are set and passed with UpdateDTO. In my opinion, there should be another one ticket for check all UpdateHadlers for MediatR. So in some cases there is Update DTO but at the same time there different Hadlers which utilize initial DTO.  So the matter in another ticket.